### ISSUE TITLE
Can display pdf?

### ISSUE THREAD

**Issue by zhiephie** [wajahatkarim3/EasyFlipViewPager#5]

**Comment by wajahatkarim3** [comment]
Very nice idea. Will look into it and see if it can be done in next version release. Thank you 👍

**Comment by zhiephie** [comment]
Welcome

**Comment by iamdeadman** [comment]
Yeah, this would be a nice enough feature to add to the stack. 
I am using this library now for displaying PDF with view pager here - [https://github.com/voghDev/PdfViewPager](https://github.com/voghDev/PdfViewPager)

I think a lot of code here can be reused into your project to provide cool bookish animations to it

**Comment by wajahatkarim3** [comment]
Which library you are mentioning here? I think you forgot to put link maybe.

**Comment by iamdeadman** [comment]
Yeah, I am sorry I missed the link label earlier. Have updated my comment with the library link now, let me know if it helps!!

**Comment by erpriyesh** [comment]
@wajahatkarim3 can we open pdf in EasyFlipViewPager library ?

**Comment by erpriyesh** [comment]
@zhiephie @iamdeadman can you please tell is it possible to show pdf in thi library ?
